lib/modules/dosomething/dosomething_metatag/dosomething_metatag.module

```diff
@@ -116,7 +126,26 @@ function dosomething_metatag_add_metatag_image($node = NULL) {
         "content" => $image,
       ),
     );
-    drupal_add_html_head($element, 'facebook_share_image');
+    drupal_add_html_head($element, 'og:image');
+
+    $element = array(
+      '#tag' => 'meta',
+      '#attributes' => array(
+        "property" => "og:image:width",
+        "content" =>  740,
+      ),
+    );
+    drupal_add_html_head($element, 'og:image:width');
+
+    $element = array(
+      '#tag' => 'meta',
+      '#attributes' => array(
+        "property" => "og:image:height",
+        "content" =>  480,
+      ),
+    );
+    drupal_add_html_head($element, 'og:image:height');
```

Comment:
what's the deal with `740x480` does it have to be hardcoded? can it be pulled from the image, or is this always the size we want?